setup.py

```diff
@@ -126,6 +126,7 @@ def package_files(prefix: str, directory: str, extension: str):
     'mock-ssh-server==0.9.1',
     'cryptography==37.0.2',
     'pytest-httpserver>=1.0.4,<1.1',
+    'transformers>=4.11,<5',
```

Comment:
Since transformers is a conditional dependency, all of its import usages need to be conditional. See https://github.com/mosaicml/composer/blob/dev/STYLE_GUIDE.md#51-external-dependencies

Reply:
@ravi-mosaicml I've removed this from `setup.py` and caught transformers-related import errors as suggested above. Can you take a quick look at `composer/algorithms/alibi/attention_surgery_functions/__init__.py` and `composer/algorithms/alibi/alibi.py` (lines 100-117) to let me know if there are issues?